### PR TITLE
Silenced standard ouptut INFO noise.

### PR DESCRIPTION
**Pull request by MahatmaCane**

### CODE REVIEW

File: conda_rpms/build_rpm_structure.py [hunk 1]
```diff
@@ -23,7 +23,10 @@
 
 import generate
 import install as conda_install
+from logging import getLogger
 
+stdoutlog = getLogger('stdoutlog')
```
Comment (pelson) [inline conda_rpms/build_rpm_structure.py]: This definitely warrants a comment.

Comment (pelson) [inline conda_rpms/build_rpm_structure.py]: Did using the specific `fetch.start` etc. loggers work also?

**Comment by pelson** [comment]
@MahatmaCane - when you get a chance could you update this PR?

**Comment by MahatmaCane** [comment]
@pelson Yup, will do!

**Pull request merged**